cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/configure-bond0.bu

```diff
@@ -0,0 +1,50 @@
+variant: openshift
```

Comment:
This is the file that butane will use to auto-generate the machineConfig file. So when the machineConfig is generated and applied the connection profiles and the systemd service below will be

Reply:
This comment seems truncated, but it's correct so far: This is the Butane input file; the `Makefile` uses the `butane` command to produce the `MachineConfig` resource, which is that we actually reference in the `kustomization.yaml` file.